apps/genrsa.c

```diff
@@ -48,7 +48,7 @@ const OPTIONS genrsa_options[] = {
 #endif
 
     OPT_SECTION("Input"),
-    {"3", OPT_3, '-', "Use 3 for the E value"},
+    {"3", OPT_3, '-', "(deprecated) Use 3 for the E value"},
     {"F4", OPT_F4, '-', "Use F4 (0x10001) for the E value"},
     {"f4", OPT_F4, '-', "Use F4 (0x10001) for the E value"},
```

Comment:
At some point, someone will have to explain how F4 is the same as 0x10001...

Reply:
:)

Reply:
What about adjusting the text to say "Use Fermat number 4 (0x10001) for the E value" then?

Reply:
@t8m's suggestion is good.  I suggest using it or something similar.

There might be confusion around F₄ being the _fifth_ Fermat number (they start with F₀).  What about: **Use the Fermat number F4 (0x10001) for the E value** instead?

Reply:
That's why I did not say "Use fourth Fermat number" :grin: . But I am also ok with @paulidale text, sure.

Reply:
F4 is the fourth fermat number, right?

Reply:
It's the fifth 😀

F₀, F₁, F₂, F₃, F₄ are 3, 5, 17, 257, 65537 respectively.

Reply:
So yes, but Fermat Numbers count from 0, see
https://oeis.org/A000215
`Fermat numbers: a(n) = 2 ^ (2^n) + 1`

Reply:
And even you said F₀, F₁, F₂, F₃, F₄ 
Q.E..D :sweat_smile: :

Reply:
I vote for @paulidale's variant